blueprints/addon/files/.travis.yml

```diff
@@ -32,4 +32,4 @@ install:
   - bower install
 
 script:
-  - ember try $EMBER_TRY_SCENARIO test
+  - ember try $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
how about adding a small comment here that explains why the flag is used and in which cases it should be removed?